cli/module_generate/scripts/generate_stubs.py

```diff
@@ -12,6 +13,25 @@ def return_attribute(resource_name: str, attr: str) -> ast.Attribute:
         ctx=ast.Load())
 
 
+def replace_async_func(stmt: ast.AsyncFunctionDef) -> None:
+    stmt.body = [
+        ast.Raise(
+            exc=ast.Call(func=ast.Name(id='NotImplementedError', ctx=ast.Load()),
+                         args=[], 
+                         keywords=[]),
+            cause=None)
+    ]
+    stmt.decorator_list = []
+
+
+def get_final_imports(imports: List[str]) -> List[str]:
+    final_imports = []
+    for i in imports:
+        if i not in final_imports:
+            final_imports.append(i)
+    return final_imports
```

Comment:
This had to be done in the case that we are importing something with an `if` statement using the `sys` module. If that's the case, `sys` always has to be imported first.

Reply:
note: Yes, there is auto-formatting that changes things, but that part fails to do it correctly when it's not in the necessary order.